cliquet/initialization.py

```diff
@@ -375,17 +375,35 @@ def on_new_response(event):
     config.add_subscriber(on_new_response, NewResponse)
 
 
-def _filter_events(listener_func, actions, resources):
-    def wrapped(event):
+class EventsFilterActions(object):
```

Comment:
As it's a filter, what about naming them `ActionEventFilter`, and `ResourcesEventsFilter`?